packaging/container/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN dnf -y update && \
     dnf clean all && \
     pip3.11 install --no-cache-dir wheel && \
```

Comment:
*(probably not exactly related to this PR, but still worth noting)*

Why do you need `wheel` here, by the way? It shouldn't be necessary to use it directly because PEP 517 ephemeral build envs would use their own copy. Besides, it's moved into `setuptools` just last week.